### ISSUE TITLE
Nav Menu - Slide animation adding white space to page width

### ISSUE THREAD

**Issue by Chaoscontrol** [elementor/elementor#15184]
## Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

## Description

I noticed all my pages had extra white space on the right side. After several hours of debugging, found out my header had a Nav Menu, and the Slide animation was what was causing, along with having a Full Width Section. 

## Steps to reproduce

- Create a header or any page really. 
- Add a Section, make the content Full Width. Add a Nav Menu, Vertical layout. 
- Notice the page has no horizontal scroll. 
- Add Slide animation. 
- Notice how the page now has 10px horizontal scroll of white space. 

## Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a default WordPress theme active.
- [X] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Source distribution v5.7.32-35
	PHP Version: 7.3.28
	PHP Max Input Vars: 3000
	PHP Max Post Size: 256M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.7.2
	Site URL: https://anisamorridadi.com
	Home URL: https://anisamorridadi.com
	WP Multisite: No
	Max Upload Size: 256 MB
	Memory limit: 768M
	Permalink Structure: /%postname%/
	Language: en-GB
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.3.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_GB
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.77 Safari/537.36

== Active Plugins ==
	Compress JPEG & PNG images
		Version: 3.2.1
		Author: TinyPNG

	Duplicate Page
		Version: 4.4
		Author: mndpsingh287

	Elementor
		Version: 3.2.4
		Author: Elementor.com

	Elementor Pro
		Version: 3.2.2
		Author: Elementor.com

	Enhanced Media Library
		Version: 2.8.5
		Author: wpUXsolutions

	Happy Elementor Addons
		Version: 2.24.1
		Author: weDevs

	SG Optimizer
		Version: 5.7.20
		Author: SiteGround


== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible
	Happy Elementor Addons: Compatible

== Elementor Pro - Compatibility Tag ==
	
	Happy Elementor Addons: Compatible

== Elements Usage ==
	
	footer : 1
		image : 1
		social-icons : 1
		text-editor : 2
	header : 4
		icon : 2
		image : 1
		nav-menu : 2
	popup : 1
		nav-menu : 1
	section : 2
		button : 1
		counter : 3
		heading : 2
	widget : 0
		testimonial-carousel : 6
	wp-page : 6
		button : 3
		counter : 6
		divider : 1
		form : 2
		heading : 32
		html : 3
		icon : 3
		image : 37
		testimonial : 1
		testimonial-carousel : 2
		text-editor : 26


== Log ==
	
Log: showing 1 of 12021-05-28 08:34:43 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.2.2',
  'to' => '3.2.4',
)]

PHP: showing 2 of 2PHP: 2021-05-28 11:30:09 [notice X 32][/home/customer/www/anisamorridadi.com/public_html/wp-content/plugins/elementor-pro/license/updater.php::85] Undefined index: new_version [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-05-28 16:43:50 [notice X 1][/home/customer/www/anisamorridadi.com/public_html/wp-content/plugins/elementor/includes/template-library/sources/local.php::700] ob_end_clean(): failed to delete buffer. No buffer to delete [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
```
</details>

**Comment by Aviad-Herman** [comment]
Hi @Chaoscontrol 
Thanks for the bug report.
I was not able to reproduce the bug based on the instructions.

Try disabling all plugins except Elementor and Elementor pro and also use Elementor Hello theme. If the bug still occurs please send a screenshot or even a video.

Thanks

**Comment by Chaoscontrol** [comment]
I could reproduce it without issue. 
Here's the video. 
https://vimeo.com/563642228/facb6b6933

**Comment by Aviad-Herman** [comment]
OK I see now, sorry.
until we fix this issue just add the following CSS code to the Nav Menu widget:

```
selector {
    overflow-x: hidden;
}
```

![image](https://user-images.githubusercontent.com/73067562/122268885-4a0b0180-cee5-11eb-85f2-dbb1285d1a40.png)

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor Pro v3.5.0

Feel free to update

Thanks!
